io.github.mfat.tvhplayer.yml

```
@@ -0,0 +1,106 @@
+base-version: '5.15-24.08'
+command: tvhplayer
+separate-locales: false
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --device=all
+  - --socket=pulseaudio
+  - --filesystem=home
+  - --filesystem=xdg-videos
+  - --filesystem=xdg-download
+  - --talk-name=org.freedesktop.Notifications
+
+
+modules:
+  - name: ffmpeg
+    config-opts:
+      - --enable-pic
+      - --enable-shared
+      - --disable-doc
+      - --disable-static
+      - --enable-gpl
+      - --enable-libvpx
+      - --enable-libmp3lame
+      - --enable-libvorbis
+      - --enable-libopus
+    sources:
+      - type: archive
+        url: https://ffmpeg.org/releases/ffmpeg-4.4.5.tar.xz
+        sha256: f9514e0d3515aee5a271283df71636e1d1ff7274b15853bcd84e144be416ab07
+
+  - name: libdvbpsi
+    sources:
+      - type: archive
+        url: https://download.videolan.org/pub/libdvbpsi/1.3.3/libdvbpsi-1.3.3.tar.bz2
+        sha256: 02b5998bcf289cdfbd8757bedd5987e681309b0a25b3ffe6cebae599f7a00112
+
+  - name: libmpeg2
+    sources:
+      - type: archive
+        url: http://libmpeg2.sourceforge.net/files/libmpeg2-0.5.1.tar.gz
+        sha256: dee22e893cb5fc2b2b6ebd60b88478ab8556cb3b93f9a0d7ce8f3b61851871d4
+    config-opts:
+      - --enable-shared
+      - --disable-static
+
+  - name: libvlc
+    config-opts:
+      - BUILDCC=gcc
+      - --disable-a52
+      - --disable-lua
+      - --disable-qt
+      - --disable-ncurses
+      - --disable-udev
+      - --enable-dvbpsi
+      - --enable-libmpeg2
+    cleanup-commands:
+      - /app/lib/vlc/vlc-cache-gen /app/lib/vlc/plugins
+    sources:
+      - type: archive
+        url: https://get.videolan.org/vlc/3.0.21/vlc-3.0.21.tar.xz
+        sha256: 24dbbe1d7dfaeea0994d5def0bbde200177347136dbfe573f5b6a4cee25afbb0
+        x-checker-data:
+          type: anitya
+          project-id: 6504
+          stable-only: true
+          url-template: https://get.videolan.org/vlc/$version/vlc-$version.tar.xz
+      # You'll need to create these patch files
+
+
+  - pypi-dependencies.yaml # Generated file from flatpak-pip-generator
+
+  - name: tvhplayer
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/share/tvhplayer
+      - mkdir -p /app/share/tvhplayer/icons
+      - cp -r tvhplayer/* /app/share/tvhplayer/
+      - cp -r icons/* /app/share/tvhplayer/icons/
+      - install -D -m755 tvhplayer.sh /app/bin/tvhplayer
+      - install -D -m644 io.github.mfat.tvhplayer.desktop -t /app/share/applications/
+      - install -D icons/tvhplayer.svg /app/share/icons/hicolor/scalable/apps/io.github.mfat.tvhplayer.svg
+      - install -D -m644 io.github.mfat.tvhplayer.metainfo.xml -t /app/share/metainfo/
+    sources:
+      - type: git
+        url: "https://github.com/mfat/tvhplayer.git"
+        # Use either tag or commit hash
+        tag: v3.5.1  # Replace with actual tag from git describe --tags --abbrev=0
```

```suggestion
        tag: v3.5.1 
```

it's already a tag, the comment is incorrect.